tests/collection/collection-search.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { test } from '../fixtures';
+import { SearchOption } from '../models';
+
+test('Collection search metadata', async ({ collectionPage }) => {
+  await test.step(`Select "Search metadata"`, async () => {
+    await collectionPage.searchPage.clickSearchInputOption(
+      SearchOption.METADATA,
+    );
+  });
+
+  await test.step(`Query for "radio"`, async () => {
+    await collectionPage.searchPage.queryFor('radio');
+  });
+
+  await test.step(`Total result count is displayed`, async () => {
+    await collectionPage.collectionFacets.checkResultCount();
+  });
+
+  await test.step(`Results are displayed in display area - checking first item displayed`, async () => {
+    await collectionPage.infiniteScroller.checkFirstItemResult();
+  });
+});
+
+test('Collection search text contents', async ({ collectionPage }) => {
+  await test.step(`Select "Search text contents"`, async () => {
+    await collectionPage.searchPage.clickSearchInputOption(SearchOption.TEXT);
+  });
+
+  await test.step(`Query for "dragnet"`, async () => {
+    await collectionPage.searchPage.queryFor('dragnet');
+  });
+
+  await test.step(`Total result count is displayed`, async () => {
+    await collectionPage.collectionFacets.checkResultCount();
+  });
+
+  await test.step(`Results are displayed in display area - checking first item displayed`, async () => {
+    await collectionPage.infiniteScroller.checkFirstItemResult();
```

Comment:
suggestion: `check` leave vagueness, let's use another verb/wording to express what we are checking for.
```suggestion
    await collectionPage.infiniteScroller.displaysFirstResult();
```